File: backend/SApplauz.Shared/DTOs/Auth/LoginResponse.cs
using SApplauz.Shared.DTOs;

namespace SApplauz.Shared.DTOs.Auth;

public class LoginResponse
{
    public string Token { get; set; } = string.Empty;
    public DateTime Expires { get; set; }
    public UserDto User { get; set; } = null!;
}

File: packages/sapplauz_core/lib/models/login_response.dart
import 'user.dart';

class LoginResponse {
  final String token;
  final DateTime expires;
  final User user;

  LoginResponse({
    required this.token,
    required this.expires,
    required this.user,
  });

  factory LoginResponse.fromJson(Map<String, dynamic> json) {
    return LoginResponse(
      token: json['token'] as String,
      expires: DateTime.parse(json['expires'] as String),
      user: User.fromJson(json['user'] as Map<String, dynamic>),
    );
  }
}






